### PR TITLE
L2D5A2

### PR DESCRIPTION
**Pull request by evyryan**

### CODE REVIEW

File: kzz7yh-f10183/cod-ve09v2_kzz7yh-f10183.xml [hunk 1]
```diff
@@ -68,10 +68,11 @@
    </p>
             <div xml:id="kzz7yh-f10183-Dd1e118">
                <head xml:id="kzz7yh-f10183-Hd1e120">Quaestio 1</head>
+               <head type="question-title">Ostendo quod in angelis non sit aliqua realis composito ex esse et essentia</head>
```
Comment (RobbieBuhite) [inline kzz7yh-f10183/cod-ve09v2_kzz7yh-f10183.xml:71]:
```suggestion
               <head type="question-title">Ostendo quod in angelis non sit aliqua realis compositio ex esse et essentia</head>
```

File: kzz7yh-f18048/cod-ve09v2_kzz7yh-f18048.xml [hunk 2]
```diff
@@ -176,13 +177,14 @@
...
             <div xml:id="kzz7yh-f18048-Dd1e316">
                <head xml:id="kzz7yh-f18048-Hd1e318">Quaestio 2</head>
+               <head type="question-title">Utrum boni angeli per suam conversionem mueruerunt gloriam</head>
                <p xml:id="kzz7yh-f18048-d1e320">
       ¶ Quaestio II. 
-      <lb ed="#V" n="78"/>Ecundo queritur vtrum boni angeli per 
+      <lb ed="#V" n="78"/>SEcundo queritur vtrum boni angeli per 
       <lb ed="#V" n="79"/>suam conuersionem mueruerunt 
-      glo<lb ed="#V" n="80" break="no"/>riam. Et videtur quod non. Quia Magister in praesenti dist. ca. 
+      glo<lb ed="#V" n="80" break="no"/>riam. Et videtur quod non. Quia Magister in praesenti<!-- TODO Abbreviation?!--> dist. ca.
```
Comment (RobbieBuhite) [inline kzz7yh-f18048/cod-ve09v2_kzz7yh-f18048.xml:185]:
```suggestion
      glo<lb ed="#V" n="80" break="no"/>riam. Et videtur quod non. Quia Magister in praesenti dist. ca. 
```

Comment (RobbieBuhite) [reply kzz7yh-f18048/cod-ve09v2_kzz7yh-f18048.xml:185]: Not sure what you are thinking is an abbreviation?

File: kzz7yh-f18048/cod-ve09v2_kzz7yh-f18048.xml [hunk 3]
```diff
@@ -255,9 +258,10 @@
             </div>
             <div xml:id="kzz7yh-f18048-Dd1e458">
                <head xml:id="kzz7yh-f18048-Hd1e460">Quaestio 3</head>
+               <head type="question-title">Utrum illi angeli sint maiores in gloria: qui nobiliorem habent naturam</head>
                <p xml:id="kzz7yh-f18048-d1e462">
       ¶ Questio. III. 
-      <lb ed="#V" n="130"/>Tertio queritur vtrum illi 
+      <lb ed="#V" n="130"/>TErtio queritur vtrum illi 
       angeli<lb ed="#V" n="131" break="no"/>sint maiores in
```
Comment (RobbieBuhite) [inline kzz7yh-f18048/cod-ve09v2_kzz7yh-f18048.xml:265]:
```suggestion
      <lb ed="#V" n="131"/>sint maiores in 
```

File: kzz7yh-f18048/cod-ve09v2_kzz7yh-f18048.xml [hunk 4]
```diff
@@ -255,9 +258,10 @@
             </div>
             <div xml:id="kzz7yh-f18048-Dd1e458">
                <head xml:id="kzz7yh-f18048-Hd1e460">Quaestio 3</head>
+               <head type="question-title">Utrum illi angeli sint maiores in gloria: qui nobiliorem habent naturam</head>
                <p xml:id="kzz7yh-f18048-d1e462">
       ¶ Questio. III. 
-      <lb ed="#V" n="130"/>Tertio queritur vtrum illi 
+      <lb ed="#V" n="130"/>TErtio queritur vtrum illi
```
Comment (RobbieBuhite) [inline kzz7yh-f18048/cod-ve09v2_kzz7yh-f18048.xml:264]:
```suggestion
      <lb ed="#V" n="130"/>TErtio queritur vtrum illi angeli
```

File: kzz7yh-f18048/cod-ve09v2_kzz7yh-f18048.xml [hunk 5]
```diff
@@ -311,8 +315,8 @@
       na<lb ed="#V" n="29" break="no"/>turam nobiliorem habuerunt. 
     </p>
                <p xml:id="kzz7yh-f18048-d1e563">
-                  <lb ed="#V" n="30"/>Ad primum in oppositum dicendum: quod deus 
-      matri<lb ed="#V" n="31" break="no"/>sue ita speciale et priuiegiatum dedit 
+                  <lb ed="#V" n="30"/>Ad primum in oppositum dicendum: quod deus matri
+      <lb ed="#V" n="31"/>fue ita speciale et priuilegiatum dedit
```
Comment (RobbieBuhite) [inline kzz7yh-f18048/cod-ve09v2_kzz7yh-f18048.xml:319]:
```suggestion
      <lb ed="#V" n="31"/>sue ita speciale et priuilegiatum dedit 
```

**Review by RobbieBuhite — CHANGES_REQUESTED** [review]

**Review by jeffreycwitt — APPROVED** [review]
Looks good!

**Pull request merged**